### PR TITLE
360 replace personal and organisation data in test files 

### PR DESCRIPTION
**Pull request by edugomez**
#938

### CODE REVIEW

File: cove_360/fixtures/SOURCES [hunk 1]
```diff
@@ -1,12 +1,20 @@
+WellcomeTrust-grants_2_grants_cp1252.csv
```
Comment (robredpath) [inline cove_360/fixtures/SOURCES]: I would prefer to not have any organisation names in filenames at all, especially given the level of editing that we've done to the data to make it break. It's right and proper that the original source be credited in documentation but I think that it's unfair to Wellcome to have their name associated with this testing data every time anyone looks at the tests - especially as CoVE is rapidly becoming more of a community project.

**Comment by robredpath** [comment]
Thanks for taking this on, @edugomez - and good work so far! I think it's important that we remove real data about funders as well as recipients, but up to you whether we do that now or we merge this (as it's still progress) and tackle removing data about funders at another time / in another PR.

**Comment by edugomez** [comment]
> tackle removing data about funders at another time / in another PR.

@robredpath I think we could do that in this same PR

**Comment by edugomez** [comment]
@robredpath can you approve this if it looks OK?

**Comment by robredpath** [comment]
This is great, @edugomez . Just a couple of quick oversights I saw when scrolling through that I've commented on

File: cove_360/fixtures/fundingproviders-grants_malformed.json [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+            "Grant type": "Large Awards", 
+            "Full name of applicant": "Prof John Doe", 
+            "awardDate": "24/07/2014"
+        }, 
+        {
+            "recipientOrganization": [
+                {
+                    "id": "GB-COH-RC000000", 
+                    "addressLocality": "Leicester", 
+                    "name": "UNIVERSITY OF LEICESTER",
```
Comment (edugomez) [inline cove_360/fixtures/fundingproviders-grants_malformed.json]: Note to self, this needs to be changed to "University of UK"

File: cove_360/fixtures/fundingproviders-grants_malformed.json [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+            "Grant type": "Large Awards", 
+            "Full name of applicant": "Miss Jane Roe", 
+            "awardDate": "24/07/2014"
+        }, 
+        {
+            "recipientOrganization": [
+                {
+                    "id": "GB-COH-RC000000", 
+                    "addressLocality": "Leicester", 
+                    "name": "UNIVERSITY OF LEICESTER",
```
Comment (edugomez) [inline cove_360/fixtures/fundingproviders-grants_malformed.json]: Note to self, this needs to be changed to "University of UK"

File: cove_360/fixtures/fundingproviders-grants_malformed.json (hunk 2 above)
Comment (edugomez) [inline cove_360/fixtures/fundingproviders-grants_malformed.json]: Note to self, this needs to be changed to "University of UK"

File: cove_360/fixtures/fundingproviders-grants_malformed.json (hunk 3 above)
Comment (edugomez) [inline cove_360/fixtures/fundingproviders-grants_malformed.json]: Note to self, this needs to be changed to "University of UK"

**Comment by robredpath** [comment]
```
Rob-Redpath-MacBook-Pro:cove robredpath$ grep -rin wellcome .
./cove_360/tests.py:98:                                         'name': 'The Wellcome Trust'}],
./cove_360/tests.py:101:                'id': '360G-wellcometrust-105183/Z/14/Z',
```

One more!

**Comment by edugomez** [comment]
@robredpath The two references you've found have been replaced. 

There are still some references to real people/organisations in flatten-tool, I've opened an issue there https://github.com/OpenDataServices/flatten-tool/issues/183 to address that.

File: cove_360/fixtures/SOURCES [hunk 4]
```diff
@@ -1,12 +1,20 @@
...
+(personal and organization indentifiers replaced)
 License: Creative Commons Attribution
 
-paul-hamlyn-foundation-grants_dc.txt
-Source: Edited version of data taken from http://data.threesixtygiving.org/sites/default/files/PHF-201314grants_1.csv
-License: Creative Commons Attribution
+tenders_releases_2_releases.xlsx
+Source: https://github.com/open-contracting/sample-data/tree/958fc30e827e980e750459e1b9a6fe09eb266d7d/buyandsell/ocds_data
+License: http://data.gc.ca/eng/open-government-licence-canada 
+(converted to xlsx format using flatten-tool; truncated to only two releases)
```
[the remark this answers is not in the document]
Comment (edugomez) [reply cove_360/fixtures/SOURCES]: That file is used in 360 functional tests https://github.com/OpenDataServices/cove/blob/master/cove_360/tests_functional.py#L71#

It is a tender release from https://buyandsell.gc.ca/ which I _think_ we don't need to anonymize /
replace

**Pull request merged**